subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -564,6 +566,7 @@ int nrf_cloud_encode_state(uint32_t reported_state, struct nrf_cloud_data *outpu
 
 		ret += json_add_str_cs(topics_obj, JSON_KEY_DEVICE_TO_CLOUD, tx_endp.ptr);
 		ret += json_add_str_cs(topics_obj, JSON_KEY_CLOUD_TO_DEVICE, rx_endp.ptr);
+		ret += json_add_str_cs(topics_obj, JSON_KEY_DEVICE_TO_CLOUD_BATCH, batch_endp.ptr);
```

Comment:
does this need to be in the shadow?
won't it always just be the d2c topic with `/batch` added?

Reply:
Hi, it will not be in the shadow. Confirmed by @savvyintegrations. Ill update the implementation.

Reply:
Updated